deep_qa/data/tokenizers/word_processor.py

```diff
@@ -0,0 +1,72 @@
+from collections import OrderedDict
+from typing import List
+from deep_qa.data.tokenizers.word_splitter import word_splitters
+from nltk.stem import PorterStemmer
+
+class WordProcessor:
+    """
+    A WordProcessor handles the splitting of strings into words (with the use of a WordSplitter) as well as any
+    desired post-processing (e.g., stemming, filtering, etc.)
+
+    Parameters
+    ----------
+    - word_spitter_choice: str
+        The string name of the WordSplitter of choice.
+    """
+    def __init__(self, word_splitter_choice: str):
+        self.word_splitter = word_splitters[word_splitter_choice]()
+
+    def split_words(self, sentence: str) -> List[str]:
+        raise NotImplementedError
+
+
+class PassThroughWordPocessor(WordProcessor):
+    """
+    The PassThroughWordProcessor splits the string into words using the specified WordSplitter and does
+    no additional post-processing to the words.
+    """
+    def split_words(self, sentence: str) -> List[str]:
+        return self.word_splitter.split_words(sentence)
+
+
+class StemmedFilteredWordProcessor(WordProcessor):
+    """
+    The StemmedFilteredWordProcessor splits the string into words using the specified WordSplitter and
+    then filters out stop words and stems the remaining words.
+    """
+    def __init__(self, word_splitter_choice: str):
+        self.stemmer = PorterStemmer()
+        # TODO (becky): load in the proper way at some point.
+        self.stopwords = ['I', 'a', 'aboard', 'about', 'above', 'accordance', 'according', 'across', 'after',
```

Comment:
Where did this list come from?  From Tushar?  There are some odd decisions (e.g., "true", "prepatory", "irrespective").